problemtools/verifyproblem.py

```diff
@@ -1165,7 +1186,7 @@ def __str__(self) -> str:
 
 _JUNK_CASES = [
     ('an empty file', b''),
-    ('a binary file with byte values 0 up to 256', bytearray(x for x in range(256))),
+    ('a binary file with byte values 0 up to 127', bytearray(x for x in range(127))),
```

Comment:
I don't think this is the right fix -- output validators that crash on invalid UTF-8 is a potential source of judge errors that we want to catch. Better change how judge messages are read to cope with invalid UTF-8.

Reply:
Ok thinking about it, it makes sense to require that every validator should be able to cope with any garbage input.

I'll try to propose a change, give me a moment.

Reply:
I replaced this with 1024 "random" bytes (generated with a fixed seed) and fixed verifyproblem crashing due to non-unicode problems.